core/node/consistency_checker/src/lib.rs

```diff
@@ -197,7 +198,7 @@ impl LocalL1BatchCommitData {
 
         let this = Self {
             l1_batch,
-            commit_tx_hash,
+            commit_tx_hash: commit_tx_hash.expect("Commit tx hash not found in the database"),
```

Comment:
why panic?

Reply:
in this file, you just changed error handling to panic

Reply:
On line 182 I have an early return if its none. The logic change here is that we no longer Error if its null. Previously consistency chcker errored on null here. This crashes the EN after making BatchStatusUpdater insert pending tranactions. I'm not sure why it works on the main node.

Reply:
I used now some clever syntax I forgot about to avoid this expect.